modules/services/ui/src/main/java/au/com/shiftyjelly/pocketcasts/ui/helper/AppIcon.kt

```diff
@@ -97,10 +97,10 @@ class AppIcon @Inject constructor(
         ),
         PRIDE_2023(
```

Comment:
```suggestion
        PRIDE(
```

Should we apply this rename?

Reply:
Thanks, I have renamed this